### PR TITLE
fix: Invalid `ID3v2` `artwork` base64 String

### LINKED ISSUE #6: Cover not found for `.mp3` File.
### Package Version

v1.2.0

### Are the dependencies being used satisfy the minimum specification?

Yes

### Summary

User has verified that the cover art is detected in other music apps correctly, however in our app, it's not found.

Looking at the file in Mp3tag, we see the cover is labeled as "Cover (Front)", which is what we should detect and return in our ID3v2Reader, however, it isn't.

### Minimal reproducible example

n/a

### PR DESCRIPTION
**Pull request by cyanChill**
# Why

`AspecT` on Discord raised an issue with some `.mp3` files not having artwork associated with it, and showed a screenshot of the same files in a different program with artwork.

He provided one of the files with this issue and I replicated the same problem.

# How

Doing a bit of investigating, I was logging the encoding for a picture frame and saw that it was different for this track compared to the others. Then I realized the problem, in which due to the encoding, the termination of reading the description would be different.
- Previously, we determined the end of a description to be when we hit a `0` byte.
- With the different encoding, it should have been when we hit 2 consecutive `0` bytes.

I kind of missed the text in the [mutagen docs](https://mutagen-specs.readthedocs.io/en/latest/id3/id3v2.4.0-frames.html#attached-picture) on the structure of a picture frame — it notes that the description is a text string according to encoding.
- The text encoding values can be found in the [ID3v2 frame overview of the mutagen docs](https://mutagen-specs.readthedocs.io/en/latest/id3/id3v2.4.0-structure.html#id3v2-frame-overview).

Basically, I had to update the `readTilNull()` function in the `FileReader`.

# Test Plan

Test to see if we render the image on the problematic file correctly through our example apps.

# Checklist

- [x] Documentation is up to date to reflect these changes (ie: `CHANGELOG.md` & `README.md`).
- [x] Code is covered by new tests and passes existing tests.
- [x] This diff will work correctly for `pnpm rnfs-example android:prod`.

### CODE REVIEW

**Comment by cyanChill** [comment]
This closes #6.
